tests/test_tools.py

```diff
@@ -228,10 +196,9 @@ def test_replace_members_with_ids(self, client, test_client):
             signatories=['~Super_User1'],
             members=['test@mail.com', '~Test_User1', '~Another_Name1']
         ))
-
         assert posted_group
 
-        client.post_profile(openreview.Profile(
+        _ = client.post_profile(openreview.Profile(
```

Comment:
what is this doing?

Reply:
I was trying to follow the changes I was making in other places in this file. Anyway, I have reverted this too now.